kernel/driver.cc

```diff
@@ -174,6 +202,8 @@ extern "C" {
 
 void yosys_atexit()
 {
+  
+  yosys_print_trace();
```

Comment:
`yosyos_atexit` does not seem to me like the right place to print a backtrace, why this place in particular?